### ISSUE TITLE
Cordova Android Can't Build Project

### ISSUE THREAD

**Issue by betorobson** [danwilson/google-analytics-plugin#147]
I got this message, any idea?

```
What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all dependencies for configuration ':_debugCompile'.
   > Could not find any version that matches com.google.android.gms:play-services-analytics:+.
     Searched in the following locations:
         https://repo1.maven.org/maven2/com/google/android/gms/play-services-analytics/maven-metadata.xml
         https://repo1.maven.org/maven2/com/google/android/gms/play-services-analytics/
         file:/Users/roberto/android-sdk-macosx/extras/android/m2repository/com/google/android/gms/play-services-analytics/maven-metadata.xml
         file:/Users/roberto/android-sdk-macosx/extras/android/m2repository/com/google/android/gms/play-services-analytics/
     Required by:
         :android:unspecified
```

**Comment by vignesh-kumar** [comment]
have you installed play services/support library ?

**Comment by betorobson** [comment]
Yes for sure, they are already installed, must I do something in my project?
![screen shot 2015-09-09 at 6 09 02 pm](https://cloud.githubusercontent.com/assets/13212208/9774305/fc21a220-571d-11e5-9389-fa4753be3ce4.png)

**Comment by betorobson** [comment]
I found the solution, I had to remove the latest version of this plugin and install 0.7.1 version.
```
cordova plugin add cordova-plugin-google-analytics@0.7.1
````
However, that is not enough, the new version leave a difference version of google play services plugin dependence. So, I had to remove it and then install version 0.7.1 of this plugin.

So, after do a downgrade we must remove this google play services before.
